flink-doris-connector/pom.xml

```diff
@@ -217,12 +217,6 @@ under the License.
             <version>2.13.3</version>
         </dependency>
 
-        <dependency>
-            <groupId>com.google.guava</groupId>
-            <artifactId>guava</artifactId>
-            <version>31.1-jre</version>
-        </dependency>
```

Comment:
In flink's coding style, need to use `flink-shaded-guava` instead

Reply:
It is recommended to remove the verification of flink-shaded-guava first, otherwise there may be problems in other Flink versions.